src/DynamoCoreWpf/ViewModels/Core/ShortcutToolbarViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Dynamo.ViewModels;
+
+namespace Dynamo.Wpf.ViewModels.Core
+{
+    public class ShortcutToolbarViewModel : ViewModelBase
```

Comment:
Other than the property, maybe the class can be internnal, please also add summary for these